bin/desi_tile_vi

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/env python
+#
+# See top-level LICENSE.rst file for Copyright information
+#
+# -*- coding: utf-8 -*-
+
+"""
+This script is used to do a visual validation of the tiles based on the automated QA results.
+"""
+
+
+import os,sys
+import argparse
+import glob
+import numpy as np
+import multiprocessing
+from astropy.table import Table
+import fitsio
+import subprocess
+from PIL import Image
```

Comment:
PIL is not one of our standard dependencies (though it happens to be installed via some other dependency), and it appears that this import is no longer used.  Let's remove it.